src/Browscap/Generator/AbstractBuildGenerator.php

```diff
@@ -87,6 +92,20 @@ public function getLogger()
     }
 
     /**
+     * Sets the flag to collect pattern ids during this build
+     *
+     * @param bool $value
+     *
+     * @return \Browscap\Generator\AbstractBuildGenerator
+     */
+    public function setCollectPatternIds($value)
```

Comment:
Though, I disagree with fluent method interfaces, it **is** consistent with existing implementation.